AutomatedReview/OverrideARStatus.md

```diff
@@ -0,0 +1,13 @@
+## Override Pull Request Status Check
+
+When the merge of your pull request is blocked by known AR failures from target branch and the AR build failure is irrelevant to your change, you can follow below process to bypass the AR build status check for the pull request.
+
+1. Send a review request to PR's SIG chairs/co-chairs.
+2. Ask SIG chairs/co-chairs to review the failed build and confirm the AR build failure is irrelevant to the PR.
+3. If you are a SIG chairs/co-chairs, run Jenkins job https://jenkins.build.o3de.org/view/Tools/job/override-pr-status-check/ to override PR's status check rule.
```

Comment:
Minor additional step here to login. Most people will hit a 403 if their token expired.

```suggestion
3. If you are a SIG chairs/co-chairs, run Jenkins job https://jenkins.build.o3de.org/view/Tools/job/override-pr-status-check/ to override PR's status check rule. Be sure to login to https://jenkins.build.o3de.org first
```